### PR TITLE
Ui updates

### PR DESCRIPTION
**Pull request by danmitchell-**
### Context
UI review from @conordelahunty 

### Changes proposed in this pull request
As per Trello https://trello.com/c/zFDKvgzF/1911-fix-style-bugs-before-deploying-new-frontend-everywhere

### Guidance to review
@conordelahunty 👍

### CODE REVIEW

File: app/views/registers/show.html.haml [hunk 1]
```diff
@@ -65,14 +65,12 @@
...
-          %span.summary Download a copy of this register
-        #details-content-2.panel.panel-border-narrow{"aria-hidden" => "true"}
-          %p Download this data as: #{link_to 'TSV', "https://#{@register.name.parameterize}.register.gov.uk/records.tsv"}, #{link_to 'CSV', "https://#{@register.name.parameterize}.register.gov.uk/records.csv"} or as a #{link_to 'Spreadsheet', "https://#{@register.name.parameterize}.register.gov.uk/records.xlsx"}.
-          %p Preview a sample as #{link_to 'JSON', "https://#{@register.name.parameterize}.register.gov.uk/preview-records/json", target: :blank}, #{link_to 'YAML', "https://#{@register.name.parameterize}.register.gov.uk/preview-records/yaml", target: :blank} or #{link_to 'TTL', "https://#{@register.name.parameterize}.register.gov.uk/preview-records/turtle", target: :blank}.
+        %li
+          - if @total_record_count > 5000
+            download the first 5,000 records
+          - else
+            download a copy of this register
+          as a #{link_to 'CSV', "https://#{@register.name.parameterize}.register.gov.uk/records.csv"} and keep your copy up-to-date
```
Comment (gidsg) [inline app/views/registers/show.html.haml]: hardcoding to `register.gov.uk` won't work for non ready to use registers. You need to work out the url based on the phase.

Comment (gidsg) [reply app/views/registers/show.html.haml]: You want to add `?page-size=5000` to this URL.

Comment (danmitchell-) [reply app/views/registers/show.html.haml]: @gidsg I agree however I was under the impression that discovery registers now on cloudapps were "throw away" and not for the general public to view so we wouldnt show the `view register` link for those on the collection page? cc @elliecraven

**Comment by elliecraven** [comment]
confirm - discovery registers aren't intended for general public view.

File: app/views/registers/show.html.haml (hunk 1 above)
Comment (gidsg) [reply app/views/registers/show.html.haml]: We still need a solution for alpha registers, so I would suggest reading the value from the client library, as that is the 'source of truth'.

File: app/jobs/populate_register_data_in_db_job.rb [hunk 2]
```diff
@@ -10,6 +10,7 @@ def perform(register)
     begin
     # Initialize client and download / update data
       register_client = RegistersClientWrapper.registers_client.get_register(register.name.parameterize, register.register_phase.downcase, PostgresDataStore.new(register))
+      register_url = register_client.instance_variable_get(:@register_url)
```
Comment (gidsg) [inline app/jobs/populate_register_data_in_db_job.rb]: I think this should be a public method on the client library but that needs an upstream change, so this is OK for now.

Comment (danmitchell-) [reply app/jobs/populate_register_data_in_db_job.rb]: 👍

**Pull request merged**